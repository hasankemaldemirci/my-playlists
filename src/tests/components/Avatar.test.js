import React from 'react'
import { shallow } from 'enzyme'

import Avatar from '../../components/Avatar'

let wrapper

beforeEach(() => {
  wrapper = shallow(<Avatar name="test" />)
})

test('Should render Avatar correctly', () => {
  expect(wrapper).toMatchSnapshot()
})

test('Should set figure text correctly', () => {
  const figure = wrapper.find('figure').at(0)
  expect(figure.text()).toBe('T')
})
