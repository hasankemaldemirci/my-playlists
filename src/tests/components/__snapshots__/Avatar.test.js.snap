// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Avatar correctly 1`] = `
<figure
  className="user__avatar"
>
  T
</figure>
`;
